modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/components/Sidebar/SidebarFooter.tsx

```diff
@@ -39,29 +40,46 @@ const SidebarFooter: React.FC<SidebarProps> = ({expanded, onClick}) => {
 			<div className="divider divider-full" />
 
 			<div className="d-flex justify-content-end">
-				<div onClick={onClick}>
-					<ForwardIcon
-						className={classNames('forward-icon ', {
-							'forward-icon-expanded': expanded,
-						})}
-					/>
-				</div>
+				<ToolTip
+					position="right"
+					title={!expanded ? i18n.translate('Expand') : undefined}
```

Comment:
Simplify and translation is in lower case

```suggestion
                                        title={expanded ? '' : i18n.translate('expand')}
```